assets/facets.js

```diff
@@ -184,6 +184,8 @@ class FacetFiltersForm extends HTMLElement {
       sortFilterForms.forEach((form) => {
         if (!isMobile) {
           if (form.id === 'FacetSortForm' || form.id === 'FacetFiltersForm' || form.id === 'FacetSortDrawerForm') {
+            const noJsElements = document.querySelectorAll('.no-js-list');
```

Comment:
Maybe we could use a selector that isn't adding a new class just to be used/selected in JS 🤔 What do you think about something like: `('.facets-wrap .no-js[role="list"]')` ?